fedimint-client/src/client.rs

```diff
@@ -1592,6 +1594,39 @@ impl ClientContextIface for Client {
         )
         .await;
     }
+
+    async fn read_operation_active_states<'dbtx>(
```

Comment:
To decouple `ModuleNotifier` from internals of the database, these two operations were added to the `ClientContextIface`.